crates/holochain/src/core/ribosome/host_fn/block_agent.rs

```diff
@@ -97,7 +97,7 @@ mod test {
     #[tokio::test(flavor = "multi_thread")]
     #[cfg(feature = "slow_tests")]
     async fn zome_call_get_block() {
-        holochain_trace::test_run().ok();
+        hc_sleuth::init_subscriber();
```

Comment:
Does this include enabling trace logging if I want that or do I need to switch to reading sleuth output for tests that use this?

Reply:
This uses an additional tracing subscriber to build up the hc-sleuth state in memory, while still piping events to the normal log subscriber. So you only have to use `init_subscriber` when using sleuth (and without it, sleuth does nothing). And the idea is that this won't affect the standard logs at all.

aitia logs are emitted as trace level by default so they won't show up in most logs.

This PR is designed to be as minimally invasive as possible. The intention is that without this line added, the entire PR has zero effect.